sql/mob_groups.sql

```diff
@@ -12542,6 +12542,8 @@ INSERT INTO `mob_groups` VALUES (14631,830,22,1056,0,591,0,0,58,60,0);  -- Promy
 INSERT INTO `mob_groups` VALUES (14632,5842,122,7200,0,0,0,0,77,79,0); -- Nargun
 INSERT INTO `mob_groups` VALUES (14633,5843,61,0,32,4599,0,0,80,82,0);  -- Ignamoth
 
+INSERT INTO `mob_groups` VALUES (14634,2002,88,3600,128,4600,5000,0,50,53,0) -- Ankabut
```

Comment:
First, need a semicolon.

Second, can't use the same mob_pool as Huge Spider. They are not flagged as NMs and I'm sure the aggro isn't set up correctly either. You need a new mob_pool for Ankabut.

Reply:
this, but also make sure not to add a respawn time since its a lottery pop and will be handled by the script end.

Reply:
hp and mp is also normally calculated (the core will do it base don its job and mob family for you) and not hard set unless we have retail data telling its it also has this amount